src/domain/event/utils.ts

```diff
@@ -1235,6 +1248,10 @@ const getUpdateEventActionWarning = (
     return t('event.form.editButtonPanel.warningEventInPast');
   }
 
+  if (noEmailFound && action == EVENT_ACTIONS.SEND_EMAIL) {
```

Comment:
eslint warns should have triple equaliser here, `action === EVENT_ACTIONS.SEND_EMAIL`